CONTRIBUTING.md

```diff
@@ -0,0 +1,79 @@
+# CONTRIBUTING
+
+## Docker
+
+The base recipe was generated using
+[neurodocker](https://github.com/ReproNim/neurodocker) from the Makefile.
+
+If possible do not modify the Dockerfile directly but modify it via the
+Makefile.
+
+Creating the Dockerfile, building it and testing it can be done with:
+
+```bash
+make docker_test
+```
+
+The test only run examples 1, 2 and 3 from the `examples` folder.
+
+## Dependencies version
+
```

Comment:
@gdevenyi 
Can you have a look at this section that explains how to update dependencies version?

Reply:
Since you're providing a "versioned" yml file, can you add how to generate it?